sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -155,11 +158,13 @@ export class EducationProgramOfferingController extends BaseController {
       offeringId,
     );
     if (!offering) {
-      throw new UnprocessableEntityException(
+      throw new NotFoundException(
         "Not able to find a Education Program Offering associated with the current Education Program, Location and offering.",
```

Comment:
I know its not part of your change, can you please change the message from "Not able to find a Education Program" to "Not able to find an Education Program"